deploy/sql/config.js

```diff
@@ -36,5 +36,8 @@ module.exports = {
     MaxRetryAttempts: parseInt(process.env.RETRY_MAX_ATTEMPTS, 10) || 3,
     InitialPauseMs: parseInt(process.env.RETRY_PAUSE_MS, 10) || 5000,
     PauseMultiplier: parseFloat(process.env.RETRY_PAUSE_MULTIPLIER) || 1.5
+  },
+  DummyData: {
```

Comment:
I think I would prefer this to be in the main config as this is not strictly speaking SQL config.

Reply:
it is in the main config file, for the sql app.